civihr_employee_portal/js/reports.js

```diff
@@ -792,11 +792,26 @@
           }
         });
       })
-      .controller('FiltersController', function () {
+      .controller('FiltersController', function (AbsencePeriod) {
         this.format = 'dd/MM/yyyy';
         this.placeholderFormat = 'dd/MM/yyyy';
         this.date = new Date();
         this.filtersCollapsed = true;
+      })
+      .service('AbsencePeriod', function () {
+        return {
+          getCurrent: function () {
```

Comment:
Even it might be a temporary solution please provide a documentation to the function.